core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -4,30 +4,29 @@
   <a ng-href="<[$ctrl.getTopicPageUrl()]>" class="protractor-test-topic-link">
     <div class="topic-thumbnail">
       <span class="topic-coming-soon" ng-if="!$ctrl.isPublished()" translate="I18N_CLASSROOM_PAGE_COMING_SOON"></span>
-      <picture class="fx-row" ng-class="{'grayscale-effect': !$ctrl.isPublished(), 'margin-top-offset': !$ctrl.isPublished()}">
+      <picture class="fx-row fx-main-center" ng-class="{'grayscale-effect': !$ctrl.isPublished(), 'margin-top-offset': !$ctrl.isPublished()}">
         <img ng-src="<[$ctrl.thumbnailUrl]>" alt="" class="img-thumbnail" ng-style="{background: $ctrl.getTopicSummary().getThumbnailBgColor()}">
       </picture>
     </div>
     <div class="topic-details" ng-style="{'background-color': $ctrl.getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !$ctrl.isPublished()}">
       <span class="topic-name"> <[$ctrl.getTopicSummary().getName()]> </span>
-      <span class="topic-story-count">
+      <span class="topic-story-count" ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() > 0">
         <[$ctrl.getTopicSummary().getCanonicalStoryCount()]>
         <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() === 1" translate="I18N_TOPIC_VIEWER_STORY"></span>
-        <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() !== 1" translate="I18N_TOPIC_VIEWER_STORIES"></span>
       </span>
     </div>
   </a>
 </div>
 <style>
-  .faded-topic-card {
-    pointer-events: none;
-  }
-
   .topic-card .margin-top-offset {
     margin-top: -1.5em;
   }
 
   .topic-card .grayscale-effect {
+    backdrop-filter: brightness(0.5);
+  }
+
+  .grayscale-effect .img-thumbnail {
```

Comment:
Hm, I see `filter` used in 4 places [here](https://github.com/oppia/oppia/blob/develop/core/templates/components/summary-tile/topic-summary-tile.directive.html). Instead of this, can we just add the `backdrop-filter` rule below the `filter` rules in each of these 4 places?

Reply:
Discussed offline. This need not be changed IMO.